src/historywork/CheckSingleLedgerHeaderWork.cpp

```diff
@@ -84,9 +84,17 @@ CheckSingleLedgerHeaderWork::doWork()
     }
     else if (mGetLedgerFileWork->getState() != State::WORK_SUCCESS)
     {
-        CLOG_WARNING(History,
-                     "Failed to download ledger checkpoint {} from archive {}",
-                     mFt->baseName_gz(), mArchive->getName());
+        CLOG_ERROR(
+            History,
+            "Failed to download ledger checkpoint {} from archive {}. This may "
```

Comment:
I think this error should not be transient, since we wait until checkpoint is published (this code is used for health checks, not catchup). Also, could we use the "possibly corrupt archive" error message const? It could be followed by the suggestion to focus on fixing the archives  (but not other configs, like quorum set, etc)